lib/redlock/client.rb

```diff
@@ -204,11 +172,12 @@ def try_lock_instances(resource, ttl, extend)
       false
     end
 
-    def lock_instances(resource, ttl, extend)
-      value = extend ? extend.fetch(:value) : SecureRandom.uuid
+    def lock_instances(resource, ttl, options)
+      value  = options[:extend] ? options[:extend].fetch(:value) : SecureRandom.uuid
+      method = options[:extend_life] ? :extend : :lock
```

Comment:
`extend_life` is easy to confuse with `extend`... i don't quite understand the difference